source/sass/type.scss

```diff
@@ -232,29 +232,29 @@ a {
   font-style: italic;
   letter-spacing: 0;
 
-  @include scaleText(10px, 12px);
+  @include scaleText(12px);
 }
 
 .body-link {
   letter-spacing: 0;
   line-height: 1.5;
 
-  @include scaleText(14px, 16px);
+  @include scaleText(16px);
 }
 
 .link-small {
   letter-spacing: 0;
   line-height: 1.33;
   font-weight: 900;
```

Comment:
Should it be `700` since it's `bold` in Sketch?

https://developer.mozilla.org/en/docs/Web/CSS/font-weight